runtime/westend/src/weights/xcm/pallet_xcm_benchmarks_generic.rs

```diff
@@ -49,135 +50,138 @@ impl<T: frame_system::Config> WeightInfo<T> {
 	// Storage: XcmPallet SupportedVersion (r:1 w:0)
 	// Storage: XcmPallet VersionDiscoveryQueue (r:1 w:1)
 	// Storage: XcmPallet SafeXcmVersion (r:1 w:0)
-	// Storage: Dmp DownwardMessageQueueHeads (r:1 w:1)
 	// Storage: Dmp DownwardMessageQueues (r:1 w:1)
+	// Storage: Dmp DownwardMessageQueueHeads (r:1 w:1)
 	pub(crate) fn report_holding() -> Weight {
-		Weight::from_ref_time(24_686_000 as u64)
+		Weight::from_ref_time(34_852_000 as u64)
 			.saturating_add(T::DbWeight::get().reads(5 as u64))
 			.saturating_add(T::DbWeight::get().writes(3 as u64))
 	}
 	pub(crate) fn buy_execution() -> Weight {
-		Weight::from_ref_time(3_620_000 as u64)
+		Weight::from_ref_time(5_595_000 as u64)
 	}
 	// Storage: XcmPallet Queries (r:1 w:0)
 	pub(crate) fn query_response() -> Weight {
-		Weight::from_ref_time(12_946_000 as u64)
+		Weight::from_ref_time(17_809_000 as u64)
 			.saturating_add(T::DbWeight::get().reads(1 as u64))
 	}
+	// Storage: Staking Ledger (r:1 w:0)
+	// Storage: Session NextKeys (r:1 w:0)
```

Comment:
This is bad, it's looking into two different pallet's storage just to perform a `Transact`...

Reply:
Okay, jesus, I'm just going to whitelist `remark_with_event` when the `runtime-benchmarks` feature is enabled.